test.py

```diff
@@ -102,3 +102,11 @@ def test_add_op_4():
     dis_matrix = np.array(mat.matrix)
 
     assert np.array_equal(dis_matrix, [[0, 0.5353485087893664], [0.5353485087893664, 0]])
+
```

Comment:
I know this was not done previously but I prefer to use `np.isclose` and `np.all_close` with a small tolerance to avoid possible issues w/ floats.

Reply:
See [this page](https://numpy.org/devdocs/reference/generated/numpy.array_equal.html). This is floating point comparison within a threshold.